src/Workspaces/Core/Portable/FlowAnalysis/SymbolUsageAnalysis/SymbolUsageAnalysis.AnalysisData.cs

```diff
@@ -193,7 +199,7 @@ public void OnReadReferenceFound(ISymbol symbol)
                 SymbolsReadBuilder.Add(symbol);
             }
 
-            public void OnWriteReferenceFound(ISymbol symbol, IOperation operation, bool maybeWritten)
+            public void OnWriteReferenceFound(ISymbol symbol, IOperation operation, bool maybeWritten, bool isRef)
```

Comment:
could you just take in a ValueUsageInfo  instead of thse two variables?

Reply:
Possibly, but I wasn't sure how would this callsite be encoded: https://github.com/dotnet/roslyn/pull/40900/files#diff-6a1d90ea5d0ff913914c081c7a378a73R430